lib/ZnapZend.pm

```diff
@@ -168,6 +168,12 @@ my $sendRecvCleanup = sub {
     #no HUP handler in child
     $SIG{HUP} = 'IGNORE';
 
+    if ($backupSet->{zend_delay} and $backupSet->{zend_delay} > 0){
+        $self->zLog->info("waiting $backupSet->{zend_delay} seconds before sending snaps...");
+        sleep $backupSet->{zend_delay};
+        $self->zLog->info("resume sending action");
+    }
```

Comment:
you may want to add $backupSet->{src} to the log lines, so that it is clear about which backupset you are talking ...